### DISCUSSION TITLE
Your request was approved. Your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by admztech** [community/community#147202] [GitHub Education] [answered]
### Hi everyone,

My name is Adnan. I applied for the GitHub Education package as a student. My request was approved on December 14, but the benefits are still not available. I am waiting. Can anyone help me with this, please? When will the benefits be visible on the GitHub portal

**Comment by VenciPython** [comment, marked as answer]
Don't worry It takes some time for the benefits of the GitHub Education package to become visible, even after your request has been approved. This process can take a few days :))

**Reply by admztech** [reply]
Can you tell me how many days will be almost 2 day passed .

**Reply by VenciPython** [reply]
I recommend checking your email for any messages from GitHub and ensuring that all instructions have been followed. If the issue persists, please contact GitHub support for a quicker resolution. You can reach GitHub Education support here https://support.github.com/contact/education

**Reply by admztech** [reply]
My request show  aproved

**Reply by erkamtb** [reply]
Did you receive the benefits? I did at 14th December too and I'm waiting like you

**Reply by admztech** [reply]
No yet still waiting. My request was show approved. But still waiting

**Comment by showhand168** [comment]
what is Your benefits?money or other thing?

**Reply by admztech** [reply]
free education pack for students
